src/apps/Console/ConsoleFeedback.js

```diff
@@ -114,7 +115,7 @@ export default function ConsoleFeedback({
             ${textStyle('body2')}
           `}
         >
-          Please enter the corresponding parameters & permissions needed for
+          Please enter the corresponding parameters &amp; permissions needed for
```

Comment:
Curious if having `&` was causing an issue? [It is valid HTML5](https://mathiasbynens.be/notes/ambiguous-ampersands) as long as it’s not attached to a character.

Reply:
Wasn't causing an issue, I just wanted to make sure we were safe in any case! ✌️